lib/classes/Service.js

```diff
@@ -182,23 +179,18 @@ class Service {
       }
     });
```

Comment:
I've just noticed this one. I believe we should also remove above validation. One culprit is that we won't crash up front, but I think error will anyway be cleanly indicated

Reply:
I believe this too. How ever this adds additional efforts to making sure it won't break anything, so I'd leave it as is, to be refactored in future PRs.

If you want, I can add ad TODO comment to that.

Reply:
As I checked, it's safe to just remove. At least then behavior will be similar to some other more complex errors, where internals crash due to blind assumption. Still with validation we will now provide a clear hint upfront that will make understanding of error easy (and from v2 we should start crashing on validation, so obscure errors that follow validation will be gone).

See how it'll expose:

![Screenshot 2020-06-18 at 15 45 44](https://user-images.githubusercontent.com/122434/85028044-e4bdec80-b17a-11ea-8020-692c68d5f056.png)

Reply:
I checked, it catches up. Will remove it, it will make the whole `validate` method much cleaner!